Formula/mapproxy.rb

```diff
@@ -20,14 +20,9 @@ class Mapproxy < Formula
   depends_on "proj"
   depends_on "python@3.9"
 
-  resource "certifi" do
-    url "https://files.pythonhosted.org/packages/6c/ae/d26450834f0acc9e3d1f74508da6df1551ceab6c2ce0766a593362d6d57f/certifi-2021.10.8.tar.gz"
-    sha256 "78884e7c1d4b00ce3cea67b44566851c4343c120abd683433ce934a68ea58872"
-  end
-
-  resource "pyproj" do
-    url "https://files.pythonhosted.org/packages/e4/36/c151d658ca1a1ccfd5ed82ac3b41d13c36cbd0687da97ac1beaeb3381fa8/pyproj-3.3.0.tar.gz"
-    sha256 "ce8bfbc212729e9a643f5f5d77f7a93394e032eda1e2d8799ae902d08add747e"
+  resource "Pillow" do
+    url "https://files.pythonhosted.org/packages/43/6e/59853546226ee6200f9ba6e574d11604b60ad0754d2cbd1c8f3246b70418/Pillow-9.1.1.tar.gz"
+    sha256 "7502539939b53d7565f3d11d87c78e7ec900d3c72945d4ee0e2f250d598309a0"
   end
```

Comment:
This needs change in pypi_formula_mappings.json
- Pillow is already installed as formula: `depends_on "pillow"`
- `pyproj` - test shows a warning: Found libproj >=5. Using this library without pyproj is deprecated and not fully supported. Please install pyproj >= 2.